src/main/webapp/components/query-manager/query-manager.tsx

```diff
@@ -51,6 +56,15 @@ const QueryCard = (props: QueryCardProps) => {
         <Actions attributes={attributes}>
           <EditAction onEdit={handleOpen} />
           <DeleteAction />
+          <MetacardInteractionsDropdown>
+            <CompressedExportMetacardInteraction
+              resultsToExport={props.resultsToExport}
+              zipped={true}
```

Comment:
:pencil2: I think this prop should be moved directly into CompressedExportInteraction because the name implies it will be zipped